lib/evaluate/common.h

```diff
@@ -83,7 +79,7 @@ template<typename A> struct ValueWithRealFlags {
   RealFlags flags;
 };
 
-enum class Rounding { TiesToEven, ToZero, Down, Up, TiesAwayFromZero };
+ENUM_CLASS(Rounding, TiesToEven, ToZero, Down, Up, TiesAwayFromZero)
 
 #if __BYTE_ORDER__ == __ORDER_BIG_ENDIAN__
```

Comment:
Who defines __BYTE_ORDER__?  I would be nice to know that it is the host byte order and not the target byte order.  IsHostLIttleEndian is clear :)

Reply:
It comes from the host C++ compilation environment.